### ISSUE TITLE
Parameter should written to file for postprocessing

### ISSUE THREAD

**Issue by x-sven** [Tri-o-copter/Brainware#7]
Veränderbare Parameter sollten wir abspeichern.
- Sensoroffsets ins Sensorlogfile
- Brainware startup Parameter in ein "config"-file, dass beim nächsten Starten wieder verwendet werden kann

**Comment by Residuum0** [comment]
Das ganze ohne Mavlink? Fände ich generell besser, können ja erstmal
über ssh editieren und dann im Flug neustarten ;)

Am 10.08.2015 um 11:47 schrieb x-sven:

**Comment by x-sven** [comment]
Ja, gern, wichtig ist nur, dass die Parameter zusammen mit den Logfiles verfügbar sind. Mir ist nämlich gänzlich unbekannt, welche Parameter wir für die Kalibrierversuche im Labor verwendet hatten... :-(
